mongock-core/src/main/java/com/github/cloudyrock/mongock/Mongock.java

```diff
@@ -112,9 +112,11 @@ public void execute() {
   /**
    * Closes the Mongo instance used by Mongock.
    * This will close either the connection Mongock was initiated with or that which was internally created.
+   * The `mongoClientCloseable` may be null when using `mongoTemplate` instead of a `mongoClient`.
    */
   public void close() throws IOException {
-    mongoClientCloseable.close();
+    if (mongoClientCloseable != null)
+      mongoClientCloseable.close();
```

Comment:
nice. Can you wrap it in brackets, please?

Reply:
Done. I realized I should've added braces right after I created the PR. Hope it looks okay now. Thanks!